andela/learning-map-front
# #149954569 Create Add Comment Logic

#### What does this PR do?
Create add comment functionality

#### Description of Task to be completed?
Create add comment functionality

#### How should this be manually tested?
After cloning the repo

- Run yarn install
- Run yarn start
- Access the landing page via localhost:3000 and click on the LOGIN TO CONTRIBUTE button.
- Navigate to view comments
- Add a comment
- The added comment should appear on the list of comments for that contribution

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[149954569](https://www.pivotaltracker.com/story/show/149954569)

#### Screenshots (if appropriate)
<img width="650" alt="screen shot 2017-08-23 at 15 34 49" src="https://user-images.githubusercontent.com/25595450/29616041-ad44cbfe-8818-11e7-8b00-90abffc36b78.png">

#### Questions:
N/A
